analyzer/windows/modules/auxiliary/html_scrap.py

```diff
@@ -0,0 +1,112 @@
+from __future__ import absolute_import
+
+import logging
+import os
+import time
+from io import BytesIO
+from threading import Thread
+
+from lib.common.abstracts import Auxiliary
+from lib.common.results import NetlogFile
+from lib.core.config import Config
+
+log = logging.getLogger(__name__)
+
+HAVE_SELENIUM = False
+
+try:
+    from selenium import webdriver
+    from selenium.webdriver.firefox.options import Options
+
+    HAVE_SELENIUM = True
+except Exception as ex:
+    log.error(ex)
+
+__author__ = "Jonathan Abrahamy"
+__email__ = "jonathan@intezer.com"
+__version__ = "0.0.1"
+__date__ = "03MAY2022"
+
+
+class HtmlScrap(Thread, Auxiliary):
+    def __init__(self, options=None, config=None):
+        Thread.__init__(self)
+        Auxiliary.__init__(self, options, config)
+        self.config = Config(cfg='analysis.conf')
+
+        self.enabled = True
+        if 'html_scrap' in options:
+            self.enabled = options['html_scrap']
+
+        self.driver_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', '..', 'bin',
```

Comment:
can it be an absolute path?

Reply:
No, these modules are copied into the machine, so we cannot trust an absolute path